Automattic/jetpack
# Storybook: add Jetpack Spinner

#### Changes proposed in this Pull Request:

* Add existing component to Storybook.
* Add `color` param to allow overriding the color easily.
* Add tests.
* Change the default color to black.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions for Storybook:

* Go to the Storybook folder `cd projects/js-packages/storybook`
* Run `pnpm install`
* Run `pnpm run storybook:dev`
* Storybook should open in your browser.
* Go to Playground → Jetpack Spinner → Docs and play around with the spinner.

#### Screenshot

![image](https://user-images.githubusercontent.com/390760/138488382-c2cec87f-ee06-4e64-92ea-f7bf691a24cd.png)
